portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -855,6 +857,33 @@ else if (req.getAttribute("siteId")!=null){
 
 
 		String pagePopupUrl = Web.returnUrl(req, "/page/");
+
+		//SAK-29660 - Refresh tool in the LHS page menu
+		String toolUrlPrefix = ServerConfigurationService.getToolUrl();
```

Comment:
Is this a patch (you've modified)? If not, there 's not too much reason to have this comment here.